src/components/CodingAgentApp.tsx

```
@@ -896,6 +1084,15 @@ export default function CodingAgentApp() {
                               <span className="text-amber-400"> · {t("codingAgent.denials", { n: run.permissionDenials })}</span>
                             )}
                           </p>
+                          {isLive(run.status) && (
+                            <RunProgressBar
+                              estimate={estimateRunProgress(run, Date.now())}
```

ESLint

[error] Error: Cannot call impure function during render

`Date.now` is an impure function. Calling an impure function can produce unstable results that update unpredictably when the component happens to re-render. (https://react.dev/reference/rules/components-and-hooks-must-be-pure#components-and-hooks-must-be-idempotent).

/home/jailuser/git/src/components/CodingAgentApp.tsx:1089:66
  1087 |                           {isLive(run.status) && (
  1088 |                             <RunProgressBar
> 1089 |                               estimate={estimateRunProgress(run, Date.now())}
       |                                                                  ^^^^^^^^^^ Cannot call impure function
  1090 |                               color={tone.color}
  1091 |                               timeLeft={t("codingAgent.timeLeft")}
  1092 |                               testId={`coding-agent-progress-${run.id}`}

(react-hooks/purity)